dev/com.ibm.ws.io.opentracing.opentracing-api.0.30.0/bnd.bnd

```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: OpenTracing API
+Bundle-SymbolicName: com.ibm.ws.io.opentracing.opentracing-api.0.30.0
```

Comment:
I'd caution on putting the version into the bundle name. This means any uplevel also means that every dependent user has to change the bundle symbolic name. It could be fixed in a subsequent PR, but just to caution that it ties the symbolic name strongly to the version level.